### ISSUE TITLE
Hello i have issu after export bvh to vmd animtion

### ISSUE THREAD

**Issue by shoujomami** [powroupi/blender_mmd_tools#299]
Well i have motion capture tools . i did tired to use different software to change bvh to vmd and no one work perfectly so when i apply my motion either in bvh in MMM (mikumikumoving) or vmd in MMD(mikumikudancine)  i get issues. 
hand of my mmd model dont move correctly and my motion so i tired to use blender mmdtools helper i did change armature type to mmd japaneses bone names then export it when i improt the new vmd from blender to MMD there no motion applied looks like the whole file of expect is emtpy there no motion at all i wonder what the issu i have if you know how can help me please help me.

**Comment by powroupi** [comment]
You might take a look https://github.com/powroupi/blender_mmd_tools/issues/60#issuecomment-306372057 . And please note that importing/exporting VMD depends on the type of selected object ( Mesh object for shape key animation, Armature object for bone animation, Root empty object of a MMD model for Full model animations ), and several limitations:
 - Keep bone names / shape key names short (maximum 15 bytes only)
 - Bone structure compatibility (if you want to apply BVH to MMD model), some structure may not work for simple bone-to-bone mapping.
 - Be aware of import/export scale, and object's transforms. (mmd_tools only import/export Fcurve values)
 - mmd_tools will use the Name property on MMD Bone Tools panel if that property is not empty, otherwise, Blender's bone name will be used.

**Comment by 3dcinetv** [comment]
@powroupi is there a specific japanese bone names object templates (armature) created in Blender (.blend) that we can use? I think we can retarget it to another armature (full english, full ik controls, full everything bone related) with constraints, bake the animation and at export time, choose the Japanese bone template armature to export to .VMD to the MMD application.
I think this method could work if we had that japanese bone object templates already existing in .blend format. I can do the rest as a proof of concept.

**Comment by powroupi** [comment]
>is there a specific japanese bone names object templates (armature) created in Blender (.blend) that we can use?

No, but we can import standard/populer pmd/pmx models to get japanese bone names. The problem is we don't know how to map the japanese bone names to the other (custom) armature unless both of the armatures use some standard naming format.

Basically you can directly export "another armature" (full english, full ik controls, full everything bone related) with constraints to BVH (bvh exporter will bake the animation), and the rest is the same thing. (open a new empty scene, import that BVH, fill corresponding MMD bone names, adjust BVH pose, export VMD, then import VMD to a MMD model) :smile:

**Comment by 3dcinetv** [comment]
Ok thanks. I will try on this and make a video for anyone needing this info.
